Sources/Network/NetworkConfig.swift

```diff
@@ -21,12 +21,16 @@ struct NetworkConfig {
 
     private let attestation: AttestationConfig
 
+    var transportProtocol: TransportProtocol = .http
```

Comment:
grpc protocol should be used by default

Reply:
yup, this has been changed in the final draft that PR will be up soon.